tests/cpu_bugs/ipmi_to_qemu.pm

```diff
@@ -0,0 +1,97 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+#
+# Summary: switch IPMI to QEMU
```

Comment:
Please go a bit more into detail. I am interested what "switch IPMI to QEMU" actually means :)

Reply:
I will paste some of words from commit log.